presto-native-execution/presto_cpp/main/types/PrestoToVeloxQueryPlan.cpp

```diff
@@ -1229,6 +1253,27 @@ core::PlanNodePtr VeloxQueryPlanConverterBase::toVeloxQueryPlan(
     rightKeys.emplace_back(exprConverter_.toVeloxExpr(right));
   }
 
+  // Check if this is a broadcast join (REPLICATED distribution)
```

Comment:
This logic is repeated 3 times. Is it possible to extract a common function e.g. createHashJoinNode(,...) for it?